packages/react/src/components/FloatingPicker/BaseFloatingPicker.tsx

```diff
@@ -281,7 +281,7 @@ export class BaseFloatingPicker<T, P extends IBaseFloatingPickerProps<T>>
         if (
           this.props.onRemoveSuggestion &&
           this.suggestionsControl.current &&
-          this.suggestionsControl.current.hasSuggestionSelected &&
+          this.suggestionsControl.current.hasSuggestionSelected() &&
```

Comment:
Needed to make this change to run without errors; not calling the function seems like a pretty clear bug.